bridge/engine/ActiveMediaList.cpp

```diff
@@ -977,6 +977,11 @@ void ActiveMediaList::removeFromRewriteMap(size_t endpointIdHash)
     ++_ssrcMapRevision;
 }
 
+bool ActiveMediaList::isAudioLevelNeeded() const
+{
+    return _audioParticipants.size() > _audioLastN;
```

Comment:
I lot of reading of the atomic in LockFreeList but we avoid decoding opus and calculating audio level in 1-1 calls and small conferences < ~3 people.
Except for BB calls as we do not know how many are on the remote side and assume auido level is needed for correct barge-in decisions.